src/main/java/frc/robot/subsystems/scoring/AimerIOTalon.java

```diff
@@ -146,8 +218,16 @@ public void updateInputs(AimerIOInputs inputs) {
         lastPosition = encoder.getAbsolutePosition();
         inputs.aimVelocityRadPerSec = 0.0;
 
-        inputs.aimAppliedVolts = aimerLeft.getMotorVoltage().getValueAsDouble();
-        inputs.aimStatorCurrentAmps = aimerLeft.getStatorCurrent().getValueAsDouble();
-        inputs.aimSupplyCurrentAmps = aimerLeft.getSupplyCurrent().getValueAsDouble();
+        if (!motorLeftFailure || motorCheckOverriden) {
+            inputs.aimAppliedVolts = aimerLeft.getMotorVoltage().getValueAsDouble();
+            inputs.aimStatorCurrentAmps = aimerLeft.getStatorCurrent().getValueAsDouble();
+            inputs.aimSupplyCurrentAmps = aimerLeft.getSupplyCurrent().getValueAsDouble();
+        } else {
+            inputs.aimAppliedVolts = aimerRight.getMotorVoltage().getValueAsDouble();
+            inputs.aimStatorCurrentAmps = aimerLeft.getStatorCurrent().getValueAsDouble();
+            inputs.aimSupplyCurrentAmps = aimerLeft.getSupplyCurrent().getValueAsDouble();
+        }
+
+        checkForAimMotorFailure();
```

Comment:
Should we check for motor failure at the top of this function (that way we can detect failure on the same frame we set the motor values)?